src/lib/Metadata/DomMetadata.cpp

```diff
@@ -863,6 +866,30 @@ DomInfo<T>::construct() const
             entries.emplace_back("type", befriended);
         }
     }
+    if constexpr(T::isAlias())
+    {
+        auto aliased = domCorpus_.get(I_.AliasedSymbol);
+        entries.emplace_back("name", aliased.get("name"));
+        entries.emplace_back("symbol", aliased);
+        if (I_.FullyQualifiedName)
+        {
+            entries.emplace_back("qualifier", domCreate(I_.FullyQualifiedName, domCorpus_));
+        }
+    }
+    if constexpr(T::isUsing())
+    {
+        for (auto const& id : I_.UsingSymbols)
+        {
+            auto sym = domCorpus_.get(id);
+            entries.emplace_back("name", sym.get("name"));
```

Comment:
This overwrites the `name` field of the `UsingInfo`... Replace this loop with `entries.emplace_back("symbols", dom::newArray<DomSymbolArray>(I_.UsingSymbols, domCorpus_));`